main/monmon.rb

```diff
@@ -1,13 +1,22 @@
 require 'csv'
 
 balance = Hash.new(0)
+not_sup = Hash.new(0)
+CURRENCY = %w[BYN USD RUR EUR]
```

Comment:
What about giving good name for this constant?

Reply:
Like I suggested here - https://github.com/ka8725/monmon/pull/9#commitcomment-37847298